hoomd/md/EvaluatorPairExpandedGaussian.h

```diff
@@ -0,0 +1,218 @@
+#define __PAIR_EVALUATOR_EXPANDEDGAUSSIAN_H__
+
+#ifndef __HIPCC__
+#include <string>
+#endif
+
+#include "hoomd/HOOMDMath.h"
+
+/*! \file EvaluatorPairEXPANDEDGAUSSIAN.h
+    \brief Defines the pair evaluator class for shifted Gaussian potentials
+*/
+
+// need to declare these class methods with __device__ qualifiers when building in nvcc
+// DEVICE is __host__ __device__ when included in nvcc and blank when included into the host
+// compiler
+#ifdef __HIPCC__
+#define DEVICE __device__
+#define HOSTDEVICE __host__ __device__
+#else
+#define DEVICE
+#define HOSTDEVICE
+#endif
+
+namespace hoomd
+    {
+namespace md
+    {
+//! Class for evaluating the expanded Gaussian pair potential
+/*! <b>General Overview</b>
+
+    See EvaluatorPairLJ
+
+    <b>Gauss specifics</b>
+
+    EvaluatorPairExpandedGaussian evaluates the function:
+    \f[ V_{\mathrm{expanded_gauss}}(r) = \varepsilon \exp \left[ -
+    \frac{1}{2}\left( \frac{r-\delta}}{\sigma}\right)^2 \right] \f]
+
+    The expanded Gaussian potential does not need diameter or charge. Three
+    parameters are specified and stored in a Scalar3.
+    \a epsilon is placed in \a params.x, \a sigma is in \a params.y, and
+    \a delta is in \a params.z.
+
+    \a epsilon and \a sigma are related to the standard lj parameters sigma and
+    epsilon by:
+    - \a epsilon = \f$ \varepsilon \f$
+    - \a sigma = \f$ \sigma \f$
+
+    \a delta is the shifted distance of the gaussian potential.
+
+*/
+class EvaluatorPairExpandedGaussian
+    {
+    public:
+    //! Define the parameter type used by this pair potential evaluator
+    struct param_type
+        {
+        Scalar epsilon;
+        Scalar sigma;
+        Scalar delta;
+
+        DEVICE void load_shared(char*& ptr, unsigned int& available_bytes) { }
+
+        HOSTDEVICE void allocate_shared(char*& ptr, unsigned int& available_bytes) const { }
+
+#ifdef ENABLE_HIP
+        // set CUDA memory hints
+        void set_memory_hint() const
+            {
+            // default implementation does nothing
+            }
+#endif
+
+#ifndef __HIPCC__
+        param_type() : epsilon(0), sigma(0), delta(0) { }
+
+        param_type(pybind11::dict v, bool managed = false)
+            {
+            sigma = v["sigma"].cast<Scalar>();
+            epsilon = v["epsilon"].cast<Scalar>();
+            delta = v["delta"].cast<Scalar>();
+            }
+
+        // used to facilitate unit testing
+        param_type(Scalar eps, Scalar sig, Scalar r, bool managed = false)
+            {
+            sigma = sig;
+            epsilon = eps;
+            delta = r;
+            }
+
+        pybind11::dict asDict()
+            {
+            pybind11::dict v;
+            v["sigma"] = sigma;
+            v["epsilon"] = epsilon;
+            v["delta"] = delta;
+            return v;
+            }
+#endif
+        }
+#ifdef SINGLE_PRECISION
+        __attribute__((aligned(8)));
+#else
+        __attribute__((aligned(16)));
+#endif
```

Comment:
```suggestion
        __attribute__((aligned(16)));
```
3 floats is 12 bytes - align to 16 byte boundaries.